gpu-jobs.md

```diff
@@ -34,15 +34,26 @@ capabilities (as of 3/16/2020):
 
 ## A. General Use GPUs via the CHTC GPU Lab
 
-There are a limited number of shared use GPUs available through the CHTC GPU
-Lab. Therefore, these servers have different policies about job runtimes and
-the maximum number of running jobs per user than general CHTC servers.
-These GPUs are a special investment from the UW2020 program, and the policies
-aim to maximize how many researchers can benefit from this investment.
-
-By opting-in to use the CHTC GPU Lab servers, you agree to be contacted by the
-project leaders occasionally to discuss your GPU computing and help improve the
-GPU Lab.
+CHTC has a set of GPUs that are available for use by any CHTC user with an 
+account on our high throughput computing (HTC) system. These shared use 
+GPUs form the [CHTC GPU Lab](/gpu-lab.shtml). 
+
+Because the CHTC GPU Lab is limited in size, and because the majority of 
+its GPUs were funded by a special investment from the UW2020 program, we 
+have implemented special policies to maximize how many researchers can 
+benefit from this investment. These policies are described briefly below, 
+and in greater length in the section of this guide that describes the 
+needed [submit file modifications to use the CHTC GPU Lab](#b-use-the-gpu-lab-servers-recommended).
+
+**Jobs running in the CHTC GPU Lab have different time and job number limits than normal CHTC job
+submissions**. Without any special requirements, jobs that use the CHTC GPU Lab 
+directly can run for up to 24 hours and a single user will be able to use up to 1/3 of 
```

Comment:
Maybe skip having prose, and just let this info be in the table?

Reply:
I'll make that change in my next update.